packages/justiciar/__tests__/droop.test.ts

```diff
@@ -1,10 +1,11 @@
 import { describe, expect, it } from "vitest"
 
+import { Rational } from "../src"
 import { droopQuota } from "../src/droop"
 describe(`droopQuota`, () => {
 	it(`should return the correct quota`, () => {
-		expect(droopQuota(10n, 1n)).toBe(6n)
-		expect(droopQuota(10n, 2n)).toBe(4n)
-		expect(droopQuota(10n, 3n)).toBe(3n)
+		expect(droopQuota(10n, 1n)).toStrictEqual(new Rational(10n, 2n))
```

Comment:
A plurality in a fractional vote system shouldn't have to win by a whole vote; it should have to achieve a number greater than the threshold. For a one-winner election this is >50%, for two-winner, this is >33.333333333%. Or `Rational(3n, 100n)`